images/inference-services/kfserving/run.sh

```diff
@@ -14,15 +14,40 @@ else
     export PROJECT=$(tr -dc a-z0-9 </dev/urandom | head -c 6 ; echo '')
 fi
 
-# kfserving expects the model file as model.joblib however mlflow
-# saves the model as model.pkl, so if there is no model.joblib,
-# create a copy named model.joblib from model.pkl
 mc alias set minio http://mlflow-minio:9000 ${AWS_ACCESS_KEY_ID} ${AWS_SECRET_ACCESS_KEY}
 model_bucket="minio${FUSEML_MODEL//s3:\//}"
-if ! mc ls ${model_bucket} | grep -q "model.joblib"; then
-    mc cp ${model_bucket}/model.pkl ${model_bucket}/model.joblib
+
+export PREDICTOR=${FUSEML_PREDICTOR}
```

Comment:
Who sets `FUSEML_PREDICTOR` value?

Reply:
it is one of the inputs from the `predictor` step

Reply:
Ah, and the env variable (with `FUSEML_` prefix is created automatically from all the inputs in the workflow, right? I vaguely remember now, but do we have it documented somewhere?

Reply:
correct, afaik there is not documentation about workflows yet

Reply:
OK, such discussion is out of the scope of this PR, it just shows how much hidden information we already have at this time...

Reply:
Indeed, just like anything that lacks documentation.